modules/js/states/actionResolve.js

```diff
@@ -176,6 +176,16 @@ define(["dojo", "dojo/_base/declare"], (dojo, declare) => {
           dojo.attr("button_" + choice.value, "data-value", choice.value);
         }
       },
+      buttonsRockPaperScissors: function (that, args) {
```

Comment:
At the moment, this function doesn't seem entirely necessary. 
But I guess what you had in mind was that in the future we will implement a new UI and then it will become needed?